src/main/resources/migrations.xml

```diff
@@ -869,10 +869,10 @@
 			<column name="plannedCompletion" type="datetime" />
 		</addColumn>
 		<addColumn tableName="tasks">
-			<column name="customField" type="text" />
+			<column name="customField" type="nvarchar(max)" />
```

Comment:
Has this migration already run on staging/production? Then you should provide a new migration that alters this field. You can't really modify historic migrations.